### PR TITLE
Cohort schedule, learning objectives, recording

### PR DESCRIPTION
**Pull request by bebatut**
- Update schedules
- Add learning objectives into schedule as lists (same for before and after elements)
- Display learning objectives into schedule page
- Add learning objectives to BioSchemas annotations
- Embed the recording directly in the schedule page

How to looks like for OLS-8: https://deploy-preview-693--ols-bebatut.netlify.app/openseeds/ols-8/schedule#cohort-call-welcome-to-open-seeds--tooling-for-project-design

### CODE REVIEW

File: _includes/week.md [hunk 1]
```diff
@@ -12,44 +14,85 @@
...
+
+<div class="columns">
+<div class="column is-half" markdown=1>
+
+### Talks
 
-**Talks**
 {% for r in c.talks %}
-- <i class="fas fa-file-powerpoint"></i>{% if r.link %} [{{ r.title }}]({{ r.link }}){% else %} {{ r.title }}{% endif %}{% if r.speaker %}{% if site.data.people[r.speaker].first-name %}, by [{{ site.data.people[r.speaker].first-name }} {{ site.data.people[r.speaker].last-name }}](/{{ cohort }}#{{ r.speaker }}){% endif %}{% endif %}
+- <i class="fas fa-file-powerpoint"></i>{% if r.slides %} [{% if r.title %}{{ r.title }}{% else %}{{ r.tag }}{% endif %}]({{ r.slides }}){% else %} {{ r.title }}{% endif %}{% if r.speakers %}{% assign speakers = '' %} {% for p in r.speakers %}{% capture speakers %}{{ speakers }}, [{{ site.data.people[p].first-name }} {{ site.data.people[p].last-name }}]({% link people.md %}#{{ p }}){% endcapture %}{% endfor %}, by {{ speakers | remove_first: ', ' }}{% endif %}
```
Comment (NPDebs) [inline _includes/week.md:57]: I think we can do without the ',' here. It looks better visually.

```suggestion
- <i class="fas fa-file-powerpoint"></i>{% if r.slides %} [{% if r.title %}{{ r.title }}{% else %}{{ r.tag }}{% endif %}]({{ r.slides }}){% else %} {{ r.title }}{% endif %}{% if r.speakers %}{% assign speakers = '' %} {% for p in r.speakers %}{% capture speakers %}{{ speakers }}, [{{ site.data.people[p].first-name }} {{ site.data.people[p].last-name }}]({% link people.md %}#{{ p }}){% endcapture %}{% endfor %} by {{ speakers | remove_first: ', ' }}{% endif %}
```

File: _includes/schedule.md [hunk 2]
```diff
@@ -111,6 +111,14 @@
             "email": "{{ site.email }}",
             "url": "{{ site.url }}"
         }]
+        {%- if c.learning_objectives -%}{%- assign lo-sep="" -%}
```
Comment (NPDebs) [inline _includes/schedule.md:114]: Good job adding learning objectives!

**Review by NPDebs — COMMENTED** [review]
It looks globally good. 💯👍 
I only had one tiny suggestion, mainly for aesthetic purpose.

Also, please note that **some speakers are missing**...but I realise it has nothing to do with the completeness of this PR. I just thought you should now, in case we need to nudge someone to add the missing info in the planning sheet.

**Comment by bebatut** [comment]
For speakers, I know we are missing information because I do not have their details yet

**Comment by bebatut** [comment]
We will know when we run the script to update the schedule: it raises the issues

**Pull request merged**